tests/gocase/integration/slotmigrate/slotmigrate_test.go

```diff
@@ -412,23 +412,19 @@ func TestSlotMigrateSync(t *testing.T) {
 	})
 
 	t.Run("MIGRATE - Migrate sync with (or without) all kinds of timeouts", func(t *testing.T) {
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync").Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", -1).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", 0).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", 10).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", 0.5).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", -3.14).Val())
+		slot := 1024
```

Comment:
Can we add a comment for this magic number? From the description it's to avoid slot num conflict.

Reply:
Sure. To be honest, we didn't take care of those slot distributions when writing test cases, so it's a bit confusing for using which slot now.

Reply:
The reason for this error is that the sync time is too small such as `0.5s` and CI often runs failed while local machines run successful. And the commit https://github.com/apache/incubator-kvrocks/commit/635dd311e5d07007826ae5338eae4cefb88ccc55 also improves read success rate.

Reply:
So when I use a very small time parameter causing a failure, the program calls `resume` func and returns `nil` value. Maybe we need the following test.

```go
				// for result.Val() == nil && timeout < 15 {
				// 	timeout += 5
				// 	result := rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", timeout)
				// 	require.NoError(t, result.Err())
				// }
				// 	require.Equal(t, "OK", result.Val())

```

Reply:
Yes, [8c76f33](https://github.com/apache/incubator-kvrocks/pull/1464/commits/8c76f3339b344b13a9964a075d08737c9e95bb56) this commit has increased the timeout.